python/ray/util/client/dataclient.py

```diff
@@ -109,11 +109,13 @@ def close(self) -> None:
     def _blocking_send(self, req: ray_client_pb2.DataRequest
                        ) -> ray_client_pb2.DataResponse:
         if self._in_shutdown:
+            import ray
+            ray.util.disconnect()
```

Comment:
Could we just switch these to `from ray.util import disconnect`. I'd prefer to avoid importing ray?